src/nvim/ex_cmds.c

```diff
@@ -1598,13 +1605,24 @@ char *make_filter_cmd(char *cmd, char *itmp, char *otmp)
   }
 
   if (itmp != NULL) {
-    xstrlcat(buf, " < ", len - 1);
-    xstrlcat(buf, (const char *)itmp, len - 1);
+    // xstrlcat(buf, " < ", len - 1);
+    // xstrlcat(buf, (const char *)itmp, len - 1);
+    if (is_pwsh) {
+      xstrlcat(buf, " -RedirectStandardInput ", len);
+    } else {
+      xstrlcat(buf, " < ", len);
+    }
+    xstrlcat(buf, (const char *)itmp, len);
```

Comment:
Can it be caused by the change from `len - 1` to `len` here?

This `len - 1` seem to be need for to make room for the final NUL byte: <https://github.com/neovim/neovim/commit/d97d66e173b11a26015a99bed8467081d9c8733e>

Reply:
```suggestion
    if (is_pwsh) {
      xstrlcat(buf, " -RedirectStandardInput ", len - 1);
    } else {
      xstrlcat(buf, " < ", len - 1);
    }
    xstrlcat(buf, (const char *)itmp, len - 1);
```

Reply:
I'll edit it and push again. Thanks! for finding that.